src/modules/cmd_duration.rs

```diff
@@ -0,0 +1,93 @@
+use crate::config::Config;
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+/// Outputs the time it took the last command to execute
+///
+/// Will only print if last command took more than a certain amount of time to
+/// execute. Default is two seconds, but can be set by config option `min_time`.
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("cmd_duration")?;
+    if module.config_value_bool("disabled").unwrap_or(false) {
```

Comment:
This logic is already present in `context.new_module()` for all modules. 😜
https://github.com/starship/starship/blob/f40f73dc8e26e6d8730344f4c6cbb47ee3d48342/src/context.rs#L104-L109

Reply:
d'oh